### DISCUSSION TITLE
How to show onHover docs of multiple LSPs

### DISCUSSION THREAD

**Discussion by ssiltanen** [helix-editor/helix#8872] [Q&A]
With the new multi LSP support, I have a setup where multiple LSPs provide onHover information, but `<space>+k` only shows the docs of the first LSP in the languages.toml language-servers array.

Is there a way to change which LSP output is shown or to append the docs into one?

**Comment by Nintron27** [comment]
Another point on this, if there is no docs on the first LS, it would be nice if the second LS's docs are shown. Because of this it's impossible to 'hover' tailwind classes in a svelte file.

**Comment by tcoopman** [comment]
I also noticed this and created an issue for this: https://github.com/helix-editor/helix/issues/8985

**Comment by uros-5** [comment]
I have temporary solution that I use for Tailwind with other languages. Try to build from `just-hover` [branch](https://github.com/uros-5/helix/tree/just-hover).
